services/api/src/modules/minio/minio.service.ts

```diff
@@ -1,24 +1,26 @@
 import {
   Injectable,
-  OnModuleInit,
-  NotFoundException,
   Logger,
+  NotFoundException,
+  OnModuleInit,
 } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import * as AWS from 'aws-sdk';
-import { ScalewayS3Options } from './interfaces/minio-options.interface';
 import { Readable } from 'stream';
+import { ScalewayS3Options } from './interfaces/minio-options.interface';
+import { FileService } from '@services/FileService';
 
 interface StorageFile {
   stream: Readable;
   metadata: { [key: string]: string };
   contentType?: string;
+  checksum: string;
 }
 
 @Injectable()
-export class MinioService implements OnModuleInit {
+export class MinioService implements OnModuleInit, FileService {
   private s3Client: AWS.S3;
-  private readonly bucketName: string;
+  public readonly bucketName: string;
```

Comment:
```suggestion
  private readonly bucketName: string;
```
Si tu passe par le getter tu peux pas avoir l'info??